tools/ipapy2/ipapy2_MS2_annotation.xml

```diff
@@ -0,0 +1,115 @@
+        <section name="unknown" title="unknown settings">
+            <param name="ppmunk" type="float" optional="true">
+                <label>ppm for unknown</label>
+                <help>ppm associated to the 'unknown' annotation. If not provided equal to ppm.</help>
+            </param>
+            <param name="ratiounk" type="float" value="0.5">
+                <label>isotope ratio for unknown</label>
+                <help>isotope ratio associated to the 'unknown' annotation. Default 0.5.</help>
+            </param>
+            <param name="CSunk" type="float" value="0.7">
+                <label>cosine similarity for unknown</label>
+                <help>cosine similarity score associated with the 'unknown' annotation. Default 0.7.</help>
+            </param>
+        </section>
+
+        <section name="optional_settings" title="optional settings">
+            <param name="me" type="float" value="5.48579909065e-04" label="mass of the electron">
+                <help>accurate mass of the electron. Default 5.48579909065e-04</help>
+            </param>
+            <param name="ratiosd" type="float" value="0.9" label="intensity ratio">
+                <help>acceptable ratio between predicted intensity and observed intensity of isotopes</help>
+            </param>
+            <param name="ppmthr" type="float" optional="true" label="ppm threshold">
+                <help>maximum ppm possible for the annotations. if not provided equal to 2*ppm.</help>
+            </param>
+            <param name="pRTNone" type="float" value="0.8" label="no RT factor">
+                <help>multiplicative factor for the RT if no RTrange present in the database.</help>
+            </param>
+            <param name="pRTout" type="float" value="0.4" label="outside RT factor">
+                <help>multiplicative factor for the RT if measured RT is outside the RTrange present in the database.</help>
+            </param>
+            <param name="mzdCS" type="integer" value="0" label="MS2 mz threshold">
+                <help>maximum mz difference allowed when computing cosine similarity scores. 
+                If one wants to use this parameter instead of ppmCS, this must be set to 0. Default 0.</help>
+            </param>
+            <param name="ppmCS" type="integer" value="10" label="maximum ppm for cosine similarity scores">
+                <help>maximum ppm allowed when computing cosine similarity scores. 
+                If one wants to use this parameter instead of mzdCS, this must be set to 0. Default 10.</help>
+            </param>
+            <param name="evfilt" type="select" label="same collision energy">
+                <help>If true, only spectrum acquired with the same collision energy are considered. Default value False.</help>
+                <option value="False">False</option>
+                <option value="True">True</option>
+            </param>
+        </section>
```

Comment:
Probably easier to make this a boolean and set the truevalue and falsevalue to the respective fields